schemaregistry-avro/docs/PySpark/schema-registry-example-pyspark.md

```diff
@@ -0,0 +1,100 @@
+# Spark Streaming + Azure Schema Registry for PySpark
+
+This document provides producer and consumer examples using the schema registry in PySpark. 
+Similar to the [schema registry example for scala](../schema-registry-example.md), first you need to create a schema group with a schema 
+in a schema registry and register an application in Azure Active Directory (Azure AD). Please refer to [Register a Schema](../schema-registry-example.md#register-a-schema)
+and [Azure Role-Based Access Control](../schema-registry-example.md#azure-role-based-access-control) sections for more information.
+
+
+## Example Schema
+We use the following schema in both producer and consumer examples in this document.
+```json
+{
+    "namespace": "Microsoft.Azure.Data.SchemaRegistry.example",
+    "type": "record",
+    "name": "Order",
+    "fields": [
+        {
+            "name": "id",
+            "type": "string"
+        },
+        {
+            "name": "amount",
+            "type": "double"
+        },
+        {
+            "name": "description",
+            "type": "string"
+        }
+    ]
+}
+```
+
+## Consumer Examples
+We can perform the following steps to pull data from an Eventhub instance and parse it with respect to a schema from the schema registry:
+   * Create a wrapper `from_avro` function which works properly in PySpark. 
+   * Pull data from the Eventhub instance using azure-event-hubs-spark connector.
+   * Use a property object which contains required information to connect to your schema registry.
+   * Deserialize the data using `from_avro` function.
+
+### Wrapper `from_avro` Function for PySpark
+First, we need to define a wrapper function in order to access and use the `com.microsoft.azure.schemaregistry.spark.avro.functions.from_avro` 
+from JVM properly.
+
+```python
+from pyspark.sql.column import Column, _to_java_column
+
+def from_avro(col, propMap, configMap):
```

Comment:
This doesn't seem right based on the actual definition of from_avro you defined... can you update the sample for PySpark to reflect the usage correctly?

Reply:
modfified